packages/graphql-transformers-e2e-tests/src/stringSetMutations.ts

```diff
@@ -0,0 +1,193 @@
+import { DeploymentResources } from 'graphql-transformer-core';
+
+/**
+ * Used with SearchableModelTransformer Tests
+ * Changes userItems list to stringsets in dynamo so the lambda function can support it
+ */
+
+const createStringSetVTL = `
```

Comment:
Can we generate this instead of hard coding? How does these resolvers get generated when customers use the searchable transformer?

Reply:
This is more of a custom implementation here since the the generated resolver in `@model` uses `$util.dynamodb.toDynamoDB($entry.value)` to intrinsically define the the type it should be in dynamodb based off the schema.